src/System.Runtime.Serialization.Xml/ref/System.Runtime.Serialization.Xml.csproj

```diff
@@ -15,4 +15,4 @@
     <None Include="project.json" />
   </ItemGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
-</Project>
\ No newline at end of file
+</Project>
```

Comment:
I cannot add a comment at line 5 above... does AssemblyVersion need to be updated as well?